packages/calling-component-bindings/review/stable/calling-component-bindings.api.md

```diff
@@ -143,6 +143,14 @@ export type GetCallingSelector<Component extends (props: any) => JSX.Element | u
 // @public
 export const getCallingSelector: <Component extends (props: any) => JSX.Element | undefined>(component: Component) => GetCallingSelector<Component>;
 
+// @public
+export type HoldButtonSelector = (state: CallClientState, props: CallingBaseSelectorProps) => {
```

Comment:
I guess the right thing to do would be to convert all these to `@internal` symbols and then rename them when exporting from `@azure/communication-react`.

We must have done this before. Let me check.

Reply:
Hmm.. This _is_ a first.

These are symbols that are exported by @azure/communication-react in beta builds.
But react-composite wants to use them unconditionally (both stable and beta builds).

Allowing this leads to a smaller diff between beta and stable builds of the composites.

Reply:
Should we mark it `@beta` instead of `@public`? (which I also don't like as then to make something stable a lot of signatures need updated)
But then with this implementation the export would need to be added to the `azure/communication-react` at stabilization time so changes like that would need to be made anyway... (_this particular component it doesn't matter because we don't export individual selectors but in terms of this happening again..._)

I'm torn on this, trying to think of a way it could be nicely excluded from the api file...